src/test/java/gov/noaa/pfel/erddap/dataset/EDDTableFromNcFilesTests.java

```diff
@@ -12390,13 +12390,13 @@ void testTablePseudoSourceNames() throws Throwable {
     expected = "station,parentDir,longitude,latitude,time,luckySeven,geoLatMin,globalZztop,latActualRangeMin,depthPositive,depthZztop,wd,wspdRange\n"
         +
         ",,degrees_east,degrees_north,UTC,m,degrees_north,,,,,degrees_true,\n" +
-        "41024,,-78.489,33.848,2014-01-15T00:00:00Z,7.0,33.848,NaN,33.848,down,,320,\"0.0, 27.0\"\n"
+        "41024,miniNdbc,-78.489,33.848,2014-01-15T00:00:00Z,7.0,33.848,NaN,33.848,down,,320,\"0.0, 27.0\"\n"
```

Comment:
I'm curious if this change still works on Windows, or if it was working without it. My understanding is that `parentDir` should be getting populated here with `miniNdbc`, since that's what's parsed from the parent path in `test_migrations/src/test/java/testDataset/EDDTestDataset.java`:

```
<dataVariable>
  <sourceName>***pathName,.*/([a-zA-Z]*)/NDBC_.*_met\\.nc,1</sourceName>
  <destinationName>parentDir</destinationName>
  <dataType>String</dataType>
  <addAttributes>
    <att name=\"ioos_category\">Identifier</att>
    <att name=\"long_name\">Parent Directory</att>
  </addAttributes>
</dataVariable>
```

Reply:
My machine(s) do not have miniNdbc there. I'm looking into that and a few other test failures I'm getting with these changes.

Reply:
I made a pull request against your branch. Several of these broke tests on windows. I believe I've adjusted them so they work on both.

Reply:
All of the affected tests still seem to pass for me with your changes, so I merged. Thanks!